pyomo/contrib/solver/tests/solvers/test_ipopt.py

```diff
@@ -302,6 +383,21 @@ def test_ipopt_config(self):
         self.assertFalse(solver.config.tee)
         self.assertTrue(solver.config.executable.startswith('/path'))
 
-        # Change value on a solve call
-        # model = self.create_model()
-        # result = solver.solve(model, tee=True)
+    def test_ipopt_solve(self):
+        model = self.create_model()
+        ipopt.Ipopt().solve(model)
```

Comment:
Does this test just make sure the solver runs without an exception?

Reply:
Yup! It's a gut-check test (because that model shouldn't fail to solve).